roles/matrix-bridge-appservice-slack/defaults/main.yml

```diff
@@ -82,7 +82,7 @@ matrix_appservice_slack_configuration_extension_yaml: |
     # Optional
     #matrix_admin_room: "!aBcDeF:matrix.org"
     #homeserver:
-    #    url: http://localhost:8008
+    #    url: http://localhost:{{ matrix_synapse_container_client_api_port }}
```

Comment:
This is just a comment. We can probably restore it to what it was.

Reply:
I changed it so that if someone reads the comment, he is getting pointed to the variable and takes care of it. I will remove it if you wish though